### ISSUE TITLE
Error - core.umd.js:3468 TypeError: Cannot read property 'create' of null on AgComponentFactory

### ISSUE THREAD

**Issue by ssashok10** [ceolter/ag-grid-ng2#60]
I am using ag-grid-ng2( v 6.1.x) for pagination and it works fine with Angular2 V2.0.0. However, if i go to any other Angular 2 page and click the back button on the browser, i am getting the below error.

Error 
```
 core.umd.js:3468 TypeError: Cannot read property 'create' of null
    at ViewContainerRef_.createComponent (core.umd.js:5307)
    at AgComponentFactory.createComponent (agComponentFactory.js:126)
    at CellRenderer.createComponent (agComponentFactory.js:75)
    at CellRenderer.BaseGuiComponent.init (agComponentFactory.js:176)
    at CellRenderer.init (agComponentFactory.js:62)
    at CellRendererService.useCellRenderer (cellRendererService.js:47)
    at RenderedCell.useCellRenderer (renderedCell.js:936)
    at RenderedCell.putDataIntoCell (renderedCell.js:885)
    at RenderedCell.populateCell (renderedCell.js:682)
    at RenderedCell.init (renderedCell.js:287)
```

This error happens only when i click the back button on the browser from the next page. Below is the code that i am using for cell rendering,

```
    {
            headerName: "Id", field: "TrackId", width: 110, cellRendererFramework: {
                template: '<a routerLink="/search/{{params.value}}">{{params.value }}</a>',
                moduleImports: [RouterModule]
            }
        },
```

I am not directly using agComponentFactory in my code. I am sure CellRendererFramework must be using it internally. Is there any work around for this issue?

**Comment by seanlandsman** [comment]
Please try the latest version - this issue was fixed in 6.1.1

**Comment by seanlandsman** [comment]
I believe that this should now be resolved
